src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  src: url("/fonts/Avenir.ttc"),
+    url("/fonts/Avenir-Book.ttf"),
+    url("/fonts/Avenir-Heavy.ttf");
+}
```

Comment:
https://stackoverflow.com/a/49010440

Reply:
Am, I took this approach because I found these fonts in the project folder, so I decided to use them all. So, in the future, I should add fonts via links in the head? Thanks!

Reply:
Could you please explain to me about this stuff? 
In this case, I should create another CSS file only with fonts and put it before CSS main style as well I should implement @import this fonts CSS file to the main CSS?)